doc/user_manual.tex

```diff
@@ -158,6 +158,101 @@ \section{WaveFunction} % (fold)
 
 
 
+
+\subsection{Two-Body Jastrow}
+\label{sub:two_body_jastrow}
+There is a set of classes already prepared if you want to define a Two-Body Jastrow.
+
+First of all you need to define the metric of your system, by implementing a child class of \verb+Metric+:
+\begin{lstlisting}
+class Metric{
```

Comment:
interesting, i wouldn't have thought of generalising this

Reply:
you need a class anyway because this component can be used in multiple places... therefore why not... you never know.
In any case, most likely the euclidean metric is the only one we need so far